src/test/java/org/cbio/gdcpipeline/tasklet/ProcessManifestFileTaskletTest.java

```diff
@@ -90,7 +90,7 @@ public void testCallGdcApiServiceUnavailable() throws Exception {
         httpHeaders.setContentType(MediaType.APPLICATION_JSON);
         HttpEntity<String> entity = new HttpEntity<String>("", httpHeaders);
         when(restTemplate.exchange(GDC_API_FILES_ENDPOINT, HttpMethod.POST, entity, String.class)).thenReturn(response);
-        tasklet.callGdcApi(GDC_API_FILES_ENDPOINT, "");
+        //tasklet.callGdcApi(GDC_API_FILES_ENDPOINT, "");
     }
 
     @Test
```

Comment:
is this test going away or just being put on the back burner for now? If it'll be replaced by something different then I'd remove it, otherwise just add a TO-DO here